vtexdocs/devportal
# Build/rapidoc method path

#### What is the purpose of this pull request?

To upgrade the RapiDoc version applied in order to update and style the OpenAPI documentation section that shows the corresponding REST method and path, as shown in the image below.
![image](https://user-images.githubusercontent.com/39717968/171057884-ca54a819-00c8-4322-a0db-139c4d21a50e.png)

[Further information here.](https://www.notion.so/vtexhandbook/Estilizar-Section-operation-url-RapiDoc-0f944339bc784d3f9cb6566254c57f6a)

#### What problem is this solving?

Inadequate structure and styling to follow [the expected design](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=3696%3A238901) for the section showing the REST method and the path referring to the documentation accessed by the user. Without changing the rapidoc source code as in [this PR](https://github.com/vtexdocs/RapiDoc/pull/1), other styling means such as CSS parts were not flexible enough to reach the desired look.

#### How should this be manually tested?

As the corresponding vtexdocs/RapiDoc PR hasn't been merged yet, this contribution is using a beta version that was generated to test the results of all applied changes together. Access this deploy preview and see if everything looks as expected.

#### Screenshots or example usage

https://user-images.githubusercontent.com/39717968/171060287-2a6a787a-54b7-441e-bdb0-78484897d0f3.mp4

#### Types of changes
